gen_bootloader_postinst_preinst.sh

```diff
@@ -1,30 +1,32 @@
 #!/bin/sh
 
-if ! [ -d ../boot ]; then
-  printf "Can't find boot dir. Run from debian subdir\n"
+if ! [ -d ./boot ]; then
+  printf "Can't find boot dir.\n"
   exit 1
 fi
 
```

Comment:
```suggestion
if ! [ -d ./debian ]; then
    printf "You have to checkout a raspios branch to create a kernel.\n"
    exit 1
fi
```

We want to work with the debian folder. How about check if it is there and report it to the user?

Reply:
Good idea, but let's check this before we try to build anything (thus in `update.sh`):
https://github.com/RevolutionPi/kernelbakery/pull/46/commits/133d0d6cbdc6eb3810539fc36eaff16aebaf18b7#diff-e14594f6bb9b29ac0146c04e78ae33e8e92f2f00709fefab97359dea31b94d11R67-R73